java/src/jmri/jmrit/display/CoordinateEdit.java

```diff
@@ -332,7 +332,7 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
     public void initSetLevel() {
         oldX = pl.getDisplayLevel();
         textX = new javax.swing.JLabel();
-        textX.setText("level= " + pl.getDisplayLevel());
+        textX.setText(Bundle.getMessage("Level") + " = " + pl.getDisplayLevel());
```

Comment:
I understand Level was added, but this should be: ```textX.setText(Bundle.getMessage("Level", pl.getDisplayLevel()));``` and the value in the .properties file for "Level" should be something like ```"level = {0}"```